### ISSUE TITLE
Text Commands

### ISSUE THREAD

**Issue by DemonTamer534** [ducktrshessami/discord-buttsbot#45]
Hey, i know this is probably kinda on me for having so many bots in my server but since discord has a limit to how many bots in one server can have slash commands it'd be nice to have text commands as an alternative. The bot already has the message intent enabled and discord got a lot softer on text commands recently (i got my own bot verified recently even though i stated "i use text commands" as one of the reasons for the message intent) so yeah it'd help a lot if i could also use text commands as an alternative

**Comment by ducktrshessami** [comment]
I am not aware of any limit on how many bots are allowed to have application commands. It's entirely possible you're just having client-side issues. (That being said, if there *is* one, it's genuinely ludicrous to me that you've added so many bots. Also I'd like to see a source.)
While it's true that this bot already has access to message content, and therefore I *could* add support for text commands back, I would prefer not to and to respect Discord's wishes. I have no comment on your bot's recent verification, but even if Discord *is* becoming more lax in enforcing the switch to application commands, I imagine they would still prefer they be used over text commands.
TLDR: I currently have no plans to resupport text command
